### DISCUSSION TITLE
Cluster names in Backend API

### DISCUSSION THREAD

**Discussion by smartx-usman** [OFTEIN-NET/OFTEIN-MultiTenantPortal#9] [Q&A] [answered]
@NattVi could you please clarify what these cluster names refer to in your API: k3d_c1, k3d_c2 (which one is GIST and CHULA)? Why you named like this in API?

**Comment by NattVi** [comment, marked as answer]
https://oftein-backend-deployment-pod-f5oec474zq-an.a.run.app/clusters

we changed name to chula gist um.
